### PR TITLE
Add CPU with Enable pin for the Golden Medal Exercise (project-05).

### PR DESCRIPTION
**Pull request by randomze**

### CODE REVIEW

**Comment by tcarreira** [comment]
parts with a `-` in the name behave differently during tests, so this is failing. 

Check `CPU-external.tst` for reference

**Comment by randomze** [comment]
Changed the files to CPUEn. Should work now right?

File: projects/05/CPUEn.hdl [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ * are affected by the execution of the current instruction, they commit 
+ * to their new values only in the next time step. If enable==1 then the
+ * processor functions as before. If enable==0, then the processor should
+ * not change neither its internal state (registers and PC) nor its outputs
+ * regardless of the passage of time or change in inputs. If reset==1 then the 
+ * CPU jumps to address 0 (i.e. pc is set to 0 in next time step) rather 
+ * than to the address resulting from executing the current instruction. 
+ */
+
+CHIP CPU-En {
```
Comment (tcarreira) [inline projects/05/CPUEn.hdl]: This CHIP name must match the filename.

**Comment by tcarreira** [comment]
After changing the CHIP name, it will work ;) 

ps: is this expected to be easier or more complex than `CPU.hdl`?

File: projects/05/CPUEn.hdl (hunk 1 above)
Comment (randomze) [reply projects/05/CPUEn.hdl]: Changed it, should be good now!

**Comment by randomze** [comment]
The idea is that it's a bit more complex since it's a gold medal. But the added complexity is small since it's a minor change to the already done CPU design.

**Pull request merged**